frontend/src/models/study.ts

```diff
@@ -13,7 +13,7 @@ export enum StudyStatus {
 
 export const launchStudy = async (api: DefaultApi, studyId: number, options: { preview?: boolean } = {}) => {
     const launch = await api.launchStudy({ id: studyId, preview: options.preview || false })
-    window.location.assign(launch.url!)
+    window.open(launch.url, '_blank')
     return launch
```

Comment:
Tested this in Safari/FF - firefox did fine, safari did block the initial popup. I came across [this approach](https://stackoverflow.com/a/54460042) in some digging around since its an async call. I wonder if this is a "reasonable enough" solution, or if we should rethink the approach 🤔